lib/render/leaderboard.php

```diff
@@ -301,8 +297,8 @@ function RenderTopAchieversComponent($user, array $gameTopAchievers, array $game
         echo "</td>";
 
         echo "<td>";
-        echo GetUserAndTooltipDiv($nextUser, true, iconSizeDisplayable: 16);
-        echo GetUserAndTooltipDiv($nextUser);
+        echo userAvatar($nextUser, true, iconSize: 16);
+        echo userAvatar($nextUser);
```

Comment:
I've noticed in some places you replace two calls to `GetUserAndTooltipDiv` with one call to `userAvatar`. Can't that be done here?

Reply:
Certainly, these seem to escape my eye even after the 20th regex search somehow. Thanks!